modules/styling/stories/mdx/Utilities.mdx

```diff
@@ -0,0 +1,279 @@
+import {ExampleCodeBlock, SymbolDoc} from '@workday/canvas-kit-docs';
+import {CreateStyles} from './examples/CreateStyles';
+import {CreateVars} from './examples/CreateVars';
+
+<Meta title="Styling/Utilities" />
+
+# Canvas Kit Styling Utilities
+
+## cs Prop
+
+The `cs` prop takes in a single, or an array of values that are created by the `cs` function, a
+string representing a CSS class name, or the return of the `createVars` function. It merges
+everything together and applies `className` and `style` attributes to a React element. Most of our components extend the `cs` prop so that you can statically apply styles to them.
+
+>**Important**: While the `cs` prop accepts a style object, **this will not** be considered statically styling an element and you will lose the performance benefits. By default, our components extend the `cs` prop.
+
+```tsx
+import {system} from '@workday/canvas-tokens-webs'
+import {PrimaryButton} from '@workday/canvas-kit-react/button'
+
+const styles = createStyles({ color: system.color.static.red.default });
+
+function MyComponent() {
+  return <PrimaryButton cs={styles}>Text</PrimaryButton>;
+}
+```
```

Comment:
Let's remove the `cs Prop` section now that this doc is called `Utilities`. Not sure I'd consider `cs` a utility. Also, this section already exists in Overview page.

```suggestion
```